ee_tests/src/page_objects/space_pipeline.page.ts

```diff
@@ -81,6 +81,12 @@ export class SpacePipelinePage extends AppPage {
     'Open In OpenShift Console'
   );
 
+  async ready() {
+    await super.ready();
+    // https://github.com/fabric8io/fabric8-test/issues/592
+    await browser.sleep(5000);
```

Comment:
Is there any way to remove this sleep statement?

Reply:
Maybe, but it will be resolved in separate issue (see comment before sleep statement)